intellij-setup.md

```diff
@@ -28,23 +28,27 @@ Android Studio (and various other JetBrains editors) is currently not supported.
 
 ### Installing the plugins
 
-We provide two plugins:
+To use IntelliJ with Flutter, you must install two plugins:
 
   * The `Dart` plugin offers code analysis (code validation as you type, code completions, etc.).
   * The `Flutter` plugin powers Flutter developer workflows (running, debugging, hot reload, etc.).
 
 To install the plugins:
 
-1. Open plugin preferences (**Preferences>Plugins** on macOS, **File>Settings>Plugins** on Linux)
-1. Select **"Browse repositories…"**
-1. Search for `'dart'`, select the Dart plug-in and click `install` (do not restart yet).
+1. Start IntelliJ.
+1. Open plugin preferences (**Preferences>Plugins** on macOS, **File>Settings>Plugins** on Linux).
+1. Select **Install JetBrains plugin...**, select the Dart plug-in and click `install` (do not restart yet).
   *  Note: this is pre-installed in WebStorm.
-1. Search for `'flutter'`,  select the Flutter plug-in and click `install`. 
+1. Select **Browse repositories…**,  select the Flutter plug-in and click `install`. 
 1. Click `Restart IntelliJ IDEA`.
 
-### Configuring the plugins
+After restarting, the Dart and Flutter plugins should both display in the left
+navigation panel when you create a new project.
 
-1. Open preferences (**Preferences** on macOS, **File>Settings** on Linux)
-1. Select **Languages & Frameworks>Flutter**
-1. Enter, or browse to, your Flutter SDK directory path in **Flutter SDK path**
+### Configuring the Flutter plugin
+
+1. Open preferences (**Preferences** on macOS, **File>Settings** on Linux).
+1. Select **Languages & Frameworks>Flutter**.
+1. Enter or browse to your Flutter SDK directory in **Flutter SDK path**
```

Comment:
Uh, that's what the following example is supposed to do, show what that looks like. 
I'll add a sentence.
Here's why I avoided talking about it by name. During setup users might have to think about:
- flutter/ (the Flutter SDK path)
- flutter/bin (the installation directory)
- flutter/bin/flutter (what `which flutter` outputs, the path to the executable)

Too many directories to think about! Where's that installer....?